roles/st2/tasks/user.yml

```diff
@@ -51,3 +50,10 @@
     option: ssh_key_file
     value: "{{ _user.ssh_key_file }}"
     backup: yes
+
+- name: Disable requiretty
```

Comment:
Just a preference if we move this block after related task: `user | Add system user to sudoers`.